project/src/main/webapp/script.js

```diff
@@ -68,5 +79,42 @@ function createPlaceElement(place) {
 function tryAgain() {
   document.getElementById('query-form').style.display = 'block';
   document.getElementById('results').style.display = 'none';
+  document.getElementById('waiting-message').style.display = 'block'
   document.getElementById('place').innerHTML = '';
 }
+
+
+/**
+ *  Creates a map and adds it to the page.
+ */
+function createMap() {
+  const USER_LOCATION = { lat: 32.080576, lng: 34.780641 }; //TODO(M1): change to user's location
+  var map = new google.maps.Map(document.getElementById('map-container'), {
+    center: USER_LOCATION,
+    zoom: 12,
```

Comment:
Maybe we should keep the zoom level as a variable that adds a little bit of information, for example - whether it's close or far?

Reply:
Good idea.